pkg/authenticator/api-key.go

```diff
@@ -60,19 +58,18 @@ func (aa *APIKeyAuthenticator) GetToken(freshTokenRequired bool) (string, uint64
 	tokenResponse, err := aa.authenticator.RequestToken()
 	if err != nil {
 		aa.logger.Error("Error fetching fresh token", zap.Error(err))
-		if !strings.Contains(strings.ToLower(err.Error()), "timeout") {
```

Comment:
Change this check to Client.Timeout in error message. Remove changing to lower case